pkg/defkinds/webserver/testdata/locks/definition.yml

```diff
@@ -0,0 +1,12 @@
+kind: webserver
```

Comment:
does it mean we should have a yaml file for the app (php kind) and a yaml file for the webserver ?

Reply:
Not really actually :slightly_smiling_face: For now, it'll be possible to directly specify the webserver to build (and its assets) from nodejs/php images (see my last commits):

```
# syntax=akerouanton/zbuilder:66c4004
kind: php
version: 7.4
fpm: true

webserver:
  webserver: nginx
  healthcheck: true
  assets:
    - from: /app/public
      to: /app/public
```

However I got quite a few ideas about what could be a zbuild tool with compose-level features by looking at https://github.com/docker/app, cnab.io and porter.sh :smile: I'd like to have some sort of `docker-compose build` command but with some advanced features like the ability to detect cross-zbuildfile image references to sort which images should be built first. With such feature, above example would be done in two separate zbuildfile.